src/shims-tsx.d.ts

```diff
@@ -0,0 +1,13 @@
+import Vue, { VNode } from 'vue'
+
+declare global {
```

Comment:
What is this for?

Reply:
This is create automatically when I convert the project to typescript. It can be removed I think (or moved to typings) but dont cause any harm.

Reply:
kk, we can just leave it